docs/enhancements/ipxe-host-boot.md

```diff
@@ -0,0 +1,125 @@
+---
+title: ipxe-host-boot
+authors:
+  - "@carbonin"
+  - “@mhrivnak”
+creation-date: 2022-02-22
+last-updated: 2022-02-22
+---
+
+# iPXE Host Boot
+
+## Summary
```

Comment:
- Think about proxies

Reply:
> @ohadlevy if I remember correctly iPXE doesn't support proxies right?

right. same for UEFI http boot afaik

Reply:
then we will probably need a warning in the UI right? 
Users still can download the artifacts and host them locally but it will not work directly.

Reply:
Is there somewhere in this doc we want to call this out specifically?
Maybe in "Risks and Mitigations"?

Reply:
Probably `### Implementation Details/Notes/Constraints` under a UI/UX section

Reply:
Added a section about proxy.